x509-cert/src/builder.rs

```diff
@@ -38,6 +37,23 @@ pub enum Error {
 
     /// Signing error propagated for the [`signature::Error`] type.
     Signature(signature::Error),
+
+    /// Each RelativeDistinguishedName MUST contain exactly one AttributeTypeAndValue.
+    NonUniqueRdn,
```

Comment:
Those new error entries are mostly tied to the checks run for compliance with cabf, I could probably make a subtype?